### PR TITLE
Test component SearchBar to exist in document

### PR DESCRIPTION
**Pull request by spottyzot**

### CODE REVIEW

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 1]
```diff
@@ -1,10 +1,14 @@
 <<<<<<< HEAD
+import React, { useState } from 'react';
+=======
+<<<<<<< HEAD
 import React from 'react';
 =======
 import React from 'react';
+>>>>>>> e111f43c61d1c18b30bf9a5f2e3c9b67f6060218
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:8]: what is this?

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 2]
```diff
@@ -1,10 +1,14 @@
 <<<<<<< HEAD
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:1]: Do you need to delete this?

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+import React, { useState } from 'react';
+=======
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:3]: Do you need to delete this?

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
+<<<<<<< HEAD
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:4]: I think this needs to be deleted.

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 5]
```diff
@@ -1,10 +1,14 @@
...
 import React from 'react';
 =======
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:6]: I think this needs to be deleted

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 6]
```diff
@@ -48,15 +52,24 @@ function answerModal({questionID}) {
...
+        <input name="email" />
+        <br />
+        <input name="photos" />
+
+      </form>
+    </div>
 
   )
 }
+<<<<<<< HEAD
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:70]: delete?

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 7]
```diff
@@ -48,15 +52,24 @@ function answerModal({questionID}) {
...
+
+export default AnswerModal;
+=======
 >>>>>>> main
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:74]: delete?

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 8]
```diff
@@ -48,15 +52,24 @@ function answerModal({questionID}) {
...
+>>>>>>> e111f43c61d1c18b30bf9a5f2e3c9b67f6060218
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:75]: delete?

File: client/src/QnA/Components/Modals/AnswerModal.jsx [hunk 9]
```diff
@@ -48,15 +52,24 @@ function answerModal({questionID}) {
...
+
+      </form>
+    </div>
 
   )
 }
+<<<<<<< HEAD
+
+export default AnswerModal;
+=======
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/AnswerModal.jsx:73]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 10]
```diff
@@ -1,13 +1,17 @@
 <<<<<<< HEAD
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 11]
```diff
@@ -1,13 +1,17 @@
...
+import React, { useState } from 'react';
+=======
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 12]
```diff
@@ -1,13 +1,17 @@
...
+<<<<<<< HEAD
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 13]
```diff
@@ -1,13 +1,17 @@
...
 import React from 'react';
 =======
 import React from 'react';
+>>>>>>> e111f43c61d1c18b30bf9a5f2e3c9b67f6060218
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 14]
```diff
@@ -1,13 +1,17 @@
 <<<<<<< HEAD
+import React, { useState } from 'react';
+=======
+<<<<<<< HEAD
 import React from 'react';
 =======
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 15]
```diff
@@ -40,14 +44,24 @@ function questionModal() {
...
+        <br />
 
-    </form>
+      </form>
+    </div>
 
   )
 
 }
+<<<<<<< HEAD
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 16]
```diff
@@ -40,14 +44,24 @@ function questionModal() {
...
+
+export default QuestionModal;
+=======
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 17]
```diff
@@ -40,14 +44,24 @@ function questionModal() {
...
 >>>>>>> main
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/Modals/QuestionModal.jsx [hunk 18]
```diff
@@ -40,14 +44,24 @@ function questionModal() {
...
+>>>>>>> e111f43c61d1c18b30bf9a5f2e3c9b67f6060218
```
Comment (simingyum) [inline client/src/QnA/Components/Modals/QuestionModal.jsx]: delete?

File: client/src/QnA/Components/QuestionList.jsx [hunk 19]
```diff
@@ -5,19 +5,22 @@ import axios from 'axios';
...
 import './styles.css'
+import QuestionModal from './Modals/QuestionModal.jsx';
 
 
 function QuestionList() {
   //state to consider: helpfulness state onClick, answersButton onClick count, questionButton onClick count,
   let productID = 5;
   const [questionCount, setQuestionCount] = useState(2);
   const [questions, setQuestions] = useState([]);
+<<<<<<< HEAD
```
Comment (simingyum) [inline client/src/QnA/Components/QuestionList.jsx:16]: delete?

File: client/src/QnA/Components/QuestionList.jsx [hunk 20]
```diff
@@ -5,19 +5,22 @@ import axios from 'axios';
...
+=======
```
Comment (simingyum) [inline client/src/QnA/Components/QuestionList.jsx:17]: delete?

File: client/src/QnA/Components/QuestionList.jsx [hunk 21]
```diff
@@ -5,19 +5,22 @@ import axios from 'axios';
...
 
   // const [expandStatus, setExpandStatus] = useState(false)
+>>>>>>> e111f43c61d1c18b30bf9a5f2e3c9b67f6060218
```
Comment (simingyum) [inline client/src/QnA/Components/QuestionList.jsx:20]: delete?

File: jest.config.js [hunk 22]
```diff
@@ -1,4 +1,8 @@
 module.exports = {
+<<<<<<< HEAD
```
Comment (simingyum) [inline jest.config.js]: delete?

File: jest.config.js [hunk 23]
```diff
@@ -1,4 +1,8 @@
...
+  testEnvironment: "jsdom",
+  setupFilesAfterEnv: ["<rootDir>/setupTests.js"]
+=======
```
Comment (simingyum) [inline jest.config.js]: delete?

File: jest.config.js [hunk 24]
```diff
@@ -13,4 +17,5 @@ module.exports = {
     "^config$": "<rootDir>/configs/app-config.js"
   }
 
+>>>>>>> e111f43c61d1c18b30bf9a5f2e3c9b67f6060218
```
Comment (simingyum) [inline jest.config.js]: delete?

**Review by simingyum — CHANGES_REQUESTED** [review]
There are some strange lines added in some of the file. Can you delete those?

**Comment by spottyzot** [comment]
I've reviewed the files on my end, I'm not sure why they have all of the heads/mains

**Pull request closed without merging**